src/users/users-module.ts

```diff
@@ -0,0 +1,39 @@
+import { randomUUID } from "crypto";
+import PostgresDB from "../modules/psql";
+
+class Users extends PostgresDB {
+    constructor(connectionString: string) {
+        super(connectionString);
+    }
+
+    
+    async addUser(username: string, email: string): Promise<void> {
+      // USER_TABLE environment variable contains the name of the table meant for storing user information
+      const id = randomUUID();
+      const userTable = process.env.USER_TABLE;
+      if (!userTable) {
+        throw new Error("USER_TABLE environment variable is not set.");
+      }
+      await this.addItem(userTable, {id, username, email});
+    }
+    
+    async getUser(id: number): Promise<{id: number, username: string, email: string} | null> {
+      // USER_TABLE environment variable contains the name of the table meant for storing user information
+      const userTable = process.env.USER_TABLE;
+      if(!userTable) {
+        throw new Error("USER_TABLE environment variable is not set.");
+      }
+      return await this.getItem(userTable, id) as {id: number, username: string, email: string} | null;
+    }
+
+    // Add additional methods or override existing ones as needed
+
+}
+
+// PSQL_CONNECTION_STRING environment variable contains the connection string for the PostgreSQL database
+const connectionString = process.env.PSQL_CONNECTION_STRING;
```

Comment:
this one as well...

Reply:
If connectionString is being passed to constructor, why this is necessary?

Reply:
Yes, is it better not to send any args to the constructor, and instead initialize a class variable with the connection string? I don't see any issues with either approach but what do you all feel?

Reply:
My inclination is towards making these part of constructor params and not have anything on this. you can check if that is the pattern followed in any example